.circleci/config.yml

```diff
@@ -137,7 +137,7 @@ jobs:
       - run: cd xcode && bundle exec pod install
       - run: bazel build --config=ci --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} -- //:PlayerUI //:PlayerUI-Demo //:PlayerUI_Pod
       # TODO: the timeout should be added to the test itself
-      - run: bazel test --test_timeout=600 --jobs=1 --verbose_failures --config=ci  --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} -- //:PlayerUI-Unit-Unit //:PlayerUI-UI-ViewInspectorTests //:PlayerUI-UI-XCUITests
+      - run: APPLITOOLS_API_KEY=${APPLITOOLS_KEY} APPLITOOLS_BATCH_ID=${CIRCLE_BUILD_NUM} bazel test --test_timeout=600 --jobs=1 --verbose_failures --config=ci  --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} -- //:PlayerUI-Unit-Unit //:PlayerUI-UI-ViewInspectorTests //:PlayerUI-UI-XCUITests
```

Comment:
Should set `APPLITOOLS_BATCH_ID` to `CIRCLE_SHA1` to make it work with the GH plugin:
![Screen Shot 2022-07-27 at 5 06 11 PM](https://user-images.githubusercontent.com/9255651/181372389-bb887caf-0f6d-4d17-a09e-d216dda159d0.png)

https://applitools.com/docs/topics/integrations/github-integration-ci-setup.html#how-to-prepare-your-code

Reply:
ohhh i forgot we can use that now in OSS